packages/paste-website/src/pages/form/input/index.mdx

```diff
@@ -0,0 +1,580 @@
+  <FormHelperText variant="error">Please enter a valid email</FormHelperText>
+</>`}
+</LivePreview>
+
+### Disabled input
+
+Use a disabled form field to show users that they can’t interact with the field.
+
+If you want to show information that can’t be edited, use a read-only form field or consider another way of showing static information.
+
+<LivePreview scope={{FormInput, FormLabel, FormHelperText}} language="jsx">
+  {`<>
+  <FormLabel htmlFor="email_disabled" disabled>Email Address</FormLabel>
+  <FormInput id="email_disabled" type="email" placeholder="example@twilio.com" disabled />
+  <FormHelperText>Please enter a valid email</FormHelperText>
+</>`}
+</LivePreview>
+
+### Read-only input
+
+Use a read-only form field when a field’s value can’t be changed, but users should be able to read or apply focus on the field. For example, use a read-only form field if a user needs to copy the value.
+
+<LivePreview scope={{FormInput, FormLabel, FormHelperText}} language="jsx">
+  {`<>
+  <FormLabel htmlFor="email_readonly">Email Address</FormLabel>
+  <FormInput id="email_readonly" type="email" placeholder="example@twilio.com" value="example@twilio.com" readOnly />
+  <FormHelperText>Please enter a valid email</FormHelperText>
+</>`}
+</LivePreview>
+
+## Composition notes
+
+An input field must have at least a label and an input.
+
+### Positioning form fields
+
+Stack form fields vertically with `$space-80` spacing between each field.
+
+<LivePreview scope={{FormInput, FormLabel, FormHelperText, Box}} language="jsx">
+  {`<>
+  <Box marginBottom="space80">
+    <FormLabel htmlFor="first_name">First Name</FormLabel>
+    <FormInput id="first_name" type="text" placeholder="John" />
+  </Box>
+  <Box>
+    <FormLabel htmlFor="last_name">Last Name</FormLabel>
+    <FormInput id="last_name" type="text" placeholder="Smith" />
+  </Box>
+</>`}
+</LivePreview>
+
+Avoid placing multiple form fields on the same horizontal row to help make it easier to scan a page vertically. Use the Grid component if you need to place form fields horizontally.
+
+### Labels and help text
+
+Labels should clearly describe the value being requested. They should be concise and descriptive, not instructional. To do this:
+
+- Use help text to provide instruction if needed. For example, don’t use “Enter the registration code on the back of your SIM card below” as label text. Instead, use “SIM registration code” as a label and “Find the registration code on the back of your SIM card” as help text.
+- Avoid articles ("a", "the") and punctuation. For example, use “SIM registration code” rather than “The SIM registration code”.
+
+To support internationalization, avoid starting a sentence with the label and using the field to finish it since sentence structures vary across languages. For example, use “Call log expiration date" or "How long should logs be stored?". Don’t use "Remove logs after:".
```

Comment:
The mix of quotes in this doc is bugging me a little `“` vs `"`, same with apostrophes `’` vs `'`. We should pick one. 

Personally I try to avoid the formatted ones from GSuite in favour of the default `"` `'`

Reply:
Agree on avoiding the formatted ones. I'll get those switched.